ReactiveObjC/RACTuple.h

```diff
@@ -111,15 +115,84 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable id)objectAtIndexedSubscript:(NSUInteger)idx;
 @end
 
+/// A tuple with exactly two generic values.
+@interface RACTwoTuple<__covariant First, __covariant Second> : RACTuple
```

Comment:
No worries at all about comments—thanks for contributing and reviewing!

Regarding `RACOneTuple`, I could be convinced, although I don't think I've ever used one in practice. Is there a scenario where you've used a tuple with a single element rather than just unwrapping the inner object? (dynamic length tuples don't count, e.g. `combineLatest:` with a dynamic number of parameters). I understand on the grounds of symmetry, but would air on the side of reducing API sprawl over rare/hypothetical use cases.

The same question applies for `RACZeroTuple`.